### PR TITLE
Censor more configs

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: lib/private/AppConfig.php [hunk 1]
```diff
@@ -45,13 +45,25 @@ class AppConfig implements IAppConfig {
 
 	/** @var array[] */
 	protected $sensitiveValues = [
+		'circles' => [
+			'/^local_gskey$/',
+		],
 		'external' => [
 			'/^sites$/',
 		],
+		'integration_discourse' => [
```
Comment (nickvergessen) [inline lib/private/AppConfig.php:54]: @eneiluj  do other integrations have similar things we should not put into the config report?

**Review by mejo- — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

hopefully some day can do this through info.xml

File: lib/private/AppConfig.php (hunk 1 above)
Comment (eneiluj) [reply lib/private/AppConfig.php:54]: @nickvergessen Yes => https://github.com/nextcloud/server/pull/31449
Thanks for mentioning that.

**Pull request merged**